key-derivation/src/lib.rs

```diff
@@ -5,8 +5,9 @@ use nimiq_hash::{hmac::*, sha512::Sha512Hash};
 use nimiq_keys::{Address, Ed25519PublicKey, PrivateKey};
 use nimiq_serde::Serialize;
 use regex::Regex;
+use serde::Deserialize;
```

Comment:
Instead of using `serde::Deserialize`, shouldn't we use `nimiq_serde::Deserialize`?